docs/index.html

```diff
@@ -0,0 +1,2007 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Ecosystem website</title>
+    <script type="module" src="https://1.www.s81c.com/common/carbon/web-components/version/v2.0.1/tile.min.js"></script>
+    <script type="module" src="https://1.www.s81c.com/common/carbon/web-components/version/v2.0.1/button.min.js"></script>
+    <script type="module" src="https://1.www.s81c.com/common/carbon/web-components/tag/v2/latest/list.min.js"></script>
+    <script type="module" src="https://1.www.s81c.com/common/carbon/web-components/tag/v2/latest/tag.min.js"></script>
+    <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon-for-ibm-dotcom/tag/v1/latest/plex.css" />
+    <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon/web-components/tag/v2/latest/themes.css" />
+    <link rel="stylesheet" href="https://1.www.s81c.com/common/carbon/web-components/tag/v2/latest/grid.css" />
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="cds-theme-zone-white custom-theme">
+
+      <header>
+        <div class="inside-header cds--grid layout-lead-space-fixed__container">
+          <img class="ecosystem-logo" src = "ecosystem-logo.svg" width="350px" alt="Qiskit Ecosystem"/>
```

Comment:
Also it's better to put the width via CSS rather than hardcoding like this. That allows us to better handle responsive design by using media queries. Maybe on larger screens we want bigger than 350px.

Reply:
Finally, logo's alt text should be `Qiskit Ecosystem logo`. You want it to accurately describe the image so visually impaired people understand what it is.